mwclient/client.py

```diff
@@ -28,6 +28,8 @@
 
 log = logging.getLogger(__name__)
 
+USER_AGENT = 'MwClient/{} ({})'.format(__ver__, 'https://github.com/mwclient/mwclient')
```

Comment:
Is MwClient or mwclient more correct? Everything except the user agent uses the lower-case variant.

Reply:
Good point, We should probably change that to `mwclient`, or, since it's after a period, `Mwclient` I guess.